processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -338,7 +492,7 @@ public void testGroupByWithSimpleRename()
         .setDimensions(
             Lists.<DimensionSpec>newArrayList(
                 new ExtractionDimensionSpec(
-                    "quality", "alias", new LookupExtractionFn(new MapLookupExtractor(map), false, null, true), null
+                    "quality", "alias", new LookupExtractionFn(new MapLookupExtractor(map), false, "MISSING", true), null
```

Comment:
It seems a bit confusing to have a value when it's not being used, so someone could be mislead in thinking that there is a reason for that string to exist. Maybe we should have both tests instead, one with true, "MISSING", and the other with false, null.